dpbench/benchmarks/black_scholes/black_scholes_dpnp.py

```diff
@@ -2,9 +2,9 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-import dpnp
+import dpnp as np
```

Comment:
Can we standardize on `dpnp`? I know we many to emphasize that `dpnp` is a drop-in replacement of NumPy, but importing `dpnp` (or anything else for that matter) as `np` is an anti-pattern IMO. It should be clear by reading some code what package is being used, `np` is a well understood alias of `NumPy` and using the same is confusing.

Reply:
Sergey specifically requested this change.

Reply:
I recall Sergey comments on this too. My concern here is what if both dpnp and numpy are needed.

Reply:
I asked him this question. His reply - Ideally all functions that are supported in numpy should be supported in dpnp. If something is not supported, we need to add support in dpnp. Also, the cases where we will need numpy and dpnp will be very few. For a large majority of programs out there it should just be possible to replace numpy with dpnp. dpbench should showcase this feature of a drop-in replacement for numpy.